tests/features/camel-core/src/test/java/org/apache/karaf/camel/itest/CamelCoreITest.java

```diff
@@ -16,10 +16,7 @@
 import java.util.List;
 
 import org.apache.camel.component.mock.MockEndpoint;
-import org.apache.karaf.camel.itests.AbstractCamelRouteWithBundleITest;
-import org.apache.karaf.camel.itests.AbstractCamelSingleFeatureResultFileBasedRoute;
-import org.apache.karaf.camel.itests.AbstractCamelSingleFeatureResultMockBasedRoute;
-import org.apache.karaf.camel.itests.CamelContextProvider;
+import org.apache.karaf.camel.itests.*;
```

Comment:
Please avoid wildcard Imports in the project

Reply:
Done